src/core/server/models/comment/comment.ts

```diff
@@ -1109,3 +1109,28 @@ export async function retrieveOngoingDiscussions(
 
   return results;
 }
+
+export async function retrieveFeaturedComments(
+  mongo: Db,
+  tenantID: string,
+  siteID: string,
+  limit: number
+) {
+  const $match: FilterQuery<Comment> = {
+    tenantID,
+    siteID,
+    "tags.type": GQLTAG.FEATURED,
+    status: { $in: PUBLISHED_STATUSES },
+  };
+  const results = await collection<Comment>(mongo)
```

Comment:
Not needed because `collection` already has this type.

```suggestion
  const results = await collection(mongo)
```